app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    test = split_integer(16, 4)
```

Comment:
According to the checklist, avoid creating unnecessary variables for simple tests. You can directly assert the result of `split_integer(16, 4)` without storing it in the `test` variable.